icons/square/48/grsync.png.svg

```diff
@@ -1 +0,0 @@
c -2.216 0 -4 -1.784 -4 -4 z m 0 0.5 l 0 0.5 c 0 2.216 1.784 4 4 4 l 38 0 c 2.216 0 4 -1.784 4 -4 l 0 -0.5 c 0 2.216 -1.784 4 -4 4 l -38 0 c -2.216 0 -4 -1.784 -4 -4 z" opacity="0.02"/><path d="m 1 43.25 l 0 0.25 c 0 2.216 1.784 4 4 4 l 38 0 c 2.216 0 4 -1.784 4 -4 l 0 -0.25 c 0 2.216 -1.784 4 -4 4 l -38 0 c -2.216 0 -4 -1.784 -4 -4 z" opacity="0.05"/><path d="m 1 43 l 0 0.25 c 0 2.216 1.784 4 4 4 l 38 0 c 2.216 0 4 -1.784 4 -4 l 0 -0.25 c 0 2.216 -1.784 4 -4 4 l -38 0 c -2.216 0 -4 -1.784 -4 -4 z" opacity="0.1"/></g><g><rect width="46" height="46" x="-47" y="1" rx="4" transform="matrix(0,-1,1,0,0,0)" fill="url(#linearGradient3764)" fill-opacity="1"/></g><g><g clip-path="url(#clipPath-165065027)"><g opacity="0.1"><!-- color: #679445 --><g><path d="m 23.988 9 c -8.262 0 -14.988 6.699 -14.988 14.984 c 0 8.289 6.727 15.01 14.988 15.01 c 8.262 0 15.01 -6.723 15.01 -15.01 c 0 -8.289 -6.75 -14.984 -15.01 -14.984 m 0 2 c 7.211 0 13.02 5.809 13.02 12.984 c 0 7.176 -5.813 13.02 -13.02 13.02 c -7.211 0 -12.996 -5.84 -12.996 -13.02 c 0 -7.176 5.781 -12.984 12.996 -12.984 m 0 0" stroke="none" fill-rule="nonzero" fill-opacity="0.502" fill="#000"/><path d="m 23.988 9 c -8.262 0 -14.988 6.699 -14.988 14.984 l 1.992 0 c 0 -7.176 5.781 -12.984 12.996 -12.984 c 7.211 0 13.02 5.809 13.02 12.984 l 0 0.137 c -0.078 7.125 -5.859 12.879 -13.02 12.879 l 0 1.996 c 8.262 0 15.01 -6.723 15.01 -15.01 c 0 -8.289 -6.75 -14.977 -15.01 -14.977 m 0 0" stroke="none" fill-rule="nonzero" fill-opacity="1" fill="#000"/><path d="m 9.563 20 c -0.352 1.273 -0.563 2.617 -0.563 4 c 0 1.727 0.305 3.395 0.844 4.938 l 0.156 0.063 l 6 -9 m -6.438 0" stroke="none" fill-rule="nonzero" fill-opacity="1" fill="#000"/></g></g></g></g><g><g clip-path="url(#clipPath-179324501)"><g transform="translate(0,-1)"><!-- color: #679445 --><g><path d="m 23.988 9 c -8.262 0 -14.988 6.699 -14.988 14.984 c 0 8.289 6.727 15.01 14.988 15.01 c 8.262 0 15.01 -6.723 15.01 -15.01 c 0 -8.289 -6.75 -14.984 -15.01 -14.984 m 0 2 c 7.211 0 13.02 5.809 13.02 12.984 c 0 7.176 -5.813 13.02 -13.02 13.02 c -7.211 0 -12.996 -5.84 -12.996 -13.02 c 0 -7.176 5.781 -12.984 12.996 -12.984 m 0 0" fill="#f9f9f9" stroke="none" fill-rule="nonzero" fill-opacity="0.502"/><path d="m 23.988 9 c -8.262 0 -14.988 6.699 -14.988 14.984 l 1.992 0 c 0 -7.176 5.781 -12.984 12.996 -12.984 c 7.211 0 13.02 5.809 13.02 12.984 l 0 0.137 c -0.078 7.125 -5.859 12.879 -13.02 12.879 l 0 1.996 c 8.262 0 15.01 -6.723 15.01 -15.01 c 0 -8.289 -6.75 -14.977 -15.01 -14.977 m 0 0" fill="#f9f9f9" stroke="none" fill-rule="nonzero" fill-opacity="1"/><path d="m 9.563 20 c -0.352 1.273 -0.563 2.617 -0.563 4 c 0 1.727 0.305 3.395 0.844 4.938 l 0.156 0.063 l 6 -9 m -6.438 0" fill="#f9f9f9" stroke="none" fill-rule="nonzero" fill-opacity="1"/></g></g></g></g><g><g transform="translate(0,-1004.3622)"><path d="m 1 1043.36 0 4 c 0 2.216 1.784 4 4 4 l 38 0 c 2.216 0 4 -1.784 4 -4 l 0 -4 c 0 2.216 -1.784 4 -4 4 l -38 0 c -2.216 0 -4 -1.784 -4 -4 z" opacity="0.1"/></g></g></svg>
```

Comment:
You deleted the wrong file! The one left uses the Square legacy design. I'm not sure why there was a discrepancy between the two though